actions/class.ItemImport.php

```diff
@@ -70,8 +71,21 @@ protected function getAvailableImportHandlers()
     private function replaceAvailableImportHandlers(): array
     {
         $returnValue = parent::getAvailableImportHandlers();
-
+        $hasAccess = $this->hasReadAccessByContext(
```

Comment:
Why no start this method with:

```
if (!$hasAccess) {
    return [];
}
```

Why iterate the loop if no importer should be allowed?

Reply:
I guess we no longer need this check and permission configuration (`isImporter`) - access rule for `taoItems_actions_ItemImport::index` will be enough.
WDYT @gabrielfs7 @chinnu-focaloid ?

Reply:
Done requested changes